src/R/Components/Test/ConnectionManager/ConnectionManagerViewModelTest.cs

```diff
@@ -39,8 +39,18 @@ public sealed class ConnectionManagerViewModelTest : IDisposable {
             var connection = _cmvm.LocalConnections.First();
             _cmvm.Connect(connection, true);
 
-            _cmvm.LocalConnections.Should().ContainSingle(c => c.Name == connection.Name)
-                .Which.IsConnected.Should().BeTrue();
+            var conns = _cmvm.LocalConnections.Where(c => c.Name == connection.Name);
+            conns.Should().ContainSingle();
+            var conn = conns.First();
+            conn.IsConnected.Should().BeTrue();
+            conn.IsRunning.Should().BeTrue();
+
+            conn.IsRunning = false;
```

Comment:
We have `ConnectionViewModelTest`. I think it is better to add a separate test there for the `IsConnected` / `IsRunning` property dependency.